src/System.Private.ServiceModel/src/System.Private.ServiceModel.builds

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Private.ServiceModel.csproj" />
+	<Project Include="System.Private.ServiceModel.csproj">
+      <OSGroup>Linux</OSGroup>
+	  <TargetGroup>dnxcore50</TargetGroup>
```

Comment:
My understanding now is that you only explicitly list an <OSGroup> if you need to build it differently (as in forked code) so at this point we build Linux differently (because of dependency on System.Net.Http.WinHttpHandler) and Windows (because we build a netcore50 flavor in addition to dnxcore50).  
I will be updating this PR based on my new understanding.